resources/views/tasks/index.blade.php

```diff
@@ -2,34 +2,74 @@
 
 @section('content')
 
-<!-- Bootstrap Boilerplate... -->
-
-	<div class="panel-body">
-		<!-- Display Validation Errors -->
-		@include('common.errors')
-
-		<!-- new task form -->
-		<form action="{{ url('tasks') }}" method="POST" class="form-horizontal">
-			{{ csrf_field()}}
-
-			<!-- Task Name -->
-			<div class="form-group">
-				<label for="task-name" class="col-sm-3 control-label">Task</label>
-
-				<div class="col-sm-6">
-					<input type="text" name="name" id="task-name" class="form-control">
-				</div>
-			</div>		
-
-			<!-- Add Task Button -->
-			<div class="form-group">
-				<div class="col-sm-offset-3 col-sm-6">
-					<button type="submit" class="btn btn-default">
-						<i class="fa fa-plus"></i> Add Task
-					</button>
-				</div>
-			</div>
-		</form>
-	</div>
-	<!-- TODO: Current Task -->
+    <div class="panel-body">
+        <!-- Display Validation Errors -->
+        @include('common.errors')
+
+        <!-- new task form -->
+        <form action="{{ url('tasks') }}" method="POST" class="form-horizontal">
+            {{ csrf_field() }}
+
+            <!-- Task Name -->
+            <div class="form-group">
+                <label for="task-name" class="col-sm-3 control-label">Task</label>
+
+                <div class="col-sm-6">
+                    <input type="text" name="name" id="task-name" class="form-control">
+                </div>
+            </div>      
+
+            <!-- Add Task Button -->
+            <div class="form-group">
+                <div class="col-sm-offset-3 col-sm-6">
+                    <button type="submit" class="btn btn-default">
+                        <i class="fa fa-plus"></i> Add Task
+                    </button>
+                </div>
+            </div>
+        </form>
+    </div>
+    <!-- TODO: Current Task -->
+    @if (count($tasks) > 0)
+
+        <div class="panel panel-default">
+
+            <div class="panel-heading">
+                Current Tasks
+            </div>
+
+            <div class="panel-body">
+
+                <table class="table table-striped task-table">
+
+                    <thead>
+                        <th>Task</th>
+                        <th>&nbsp</th>
+                    </thead>
+
+                    <tbody>
+
+                        @foreach ($tasks as $task)
+
+                            <tr>
+                                <td class="table-text">
+                                    <div>{{ $task->name }}</div>
+                                </td>
+
+                                <td>
+                                    <!-- DELETE BUTTON -->
+
+                                </td>
+
+                            </tr>
+                            
+                        @endforeach
+
+                    </tbody>
+
+                </table>
+
+            </div>
+
+        </div>
```

Comment:
@elsiedelapena I think it is better to lessen a lot of extra line spaces for this file